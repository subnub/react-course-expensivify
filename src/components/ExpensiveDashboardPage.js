import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";
import ExpenseSummary from "./ExpenseSummary";

const ExpensiveDashboardPage = () => (
    <div>
        <ExpenseListFilters />
        <ExpenseSummary />
        <ExpenseList />
    </div>
)

export default ExpensiveDashboardPage;