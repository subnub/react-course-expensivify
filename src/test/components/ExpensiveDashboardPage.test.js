import React from "react"
import {shallow} from "enzyme"
import ExpensiveDashboardPage from "../../components/ExpensiveDashboardPage"

test("Should render dashboard correctly", () => {
    const wrapper = shallow(<ExpensiveDashboardPage />)
    
    expect(wrapper).toMatchSnapshot();
})