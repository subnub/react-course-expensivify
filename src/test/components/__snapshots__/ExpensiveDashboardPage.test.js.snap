// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render dashboard correctly 1`] = `
<div>
  <React.Memo />
  <React.Memo />
  <React.Memo />
</div>
`;
